backend/src/main/java/com/htecgroup/skynest/exception/CustomException.java

```diff
@@ -6,13 +6,13 @@
 
 @Getter
 @AllArgsConstructor
-public class UserException extends RuntimeException {
+public class CustomException extends RuntimeException {
```

Comment:
Because lots of PR already use the UserException, it is good practice this refactoring to be done in separate PR

Reply:
Of corse. I ll redo changes. And than after it can be done in separate PR